app/controllers/templates_controller.rb

```diff
@@ -20,6 +20,30 @@ def new
     authorize @template
   end
 
+  # GET /templates/import
+  def new_import
```

Comment:
Can we re-use the existing new method and associated view, creating the option to either import an archive or create from scratch?